slothy/helper.py

```diff
@@ -1801,3 +1802,40 @@ def extract(arch: any, source: list, lbl: str, forced_loop_type: any = None) ->
                 pass
 
         raise FatalParsingException(f"Couldn't identify loop {lbl}")
+
+
+class UnknownInstruction(Exception):
+    pass
```

Comment:
Re-defining this class here is not a good idea. Can we have a single definition of that class please?